airqo-platform/AirQo-frontend
# Feature location registry

- What does this PR do?
This PR adds the location registry feature to the Netmanager product.
- What JIRA task is related to this PR?
The link to the JIRA board task is [here](https://airqoteam.atlassian.net/browse/NET-12?atlOrigin=eyJpIjoiZmRiMjYzNzQyMGMzNDBjOGFmNWJjY2RhZDE1ZTNhNjkiLCJwIjoiaiJ9).
- How do I test out this PR?
Git clone this repo `cd AirQo-frontend/locate` Then run `npm install` to get all the necessary dependencies Afterwards run `npm run local` to test out application.
Just head straight to the `Location Registry` tab of the sidebar to check out the presentational layout as seen in the screenshot below:
![image](https://user-images.githubusercontent.com/1590213/83146077-6b445880-a0fe-11ea-9dd5-160dc828430f.png)

- Which end points should are ready for testing?
There are no endpoints to test out here.
